infoblox/resource_infoblox_ip_allocation.go

```diff
@@ -103,7 +103,6 @@ func resourceIPAllocation() *schema.Resource {
 			},
 			"internal_id": {
 				Type:     schema.TypeString,
-				Optional: true,
 				Computed: true,
 				Description: "Internal ID of an object at NIOS side," +
```

Comment:
By the way, you can use raw string literals for multiline text instead of concatenating a couple of strings. 

```go
`This
is the
multiline
string`
```

https://go.dev/ref/spec#String_literals

Reply:
But taking code indentation into consideration, this will also include extra spaces into the text string, which is undesirable.